libraries/Adapters/Microsoft.Bot.Builder.Adapters.Twilio/TwilioHelper.cs

```diff
@@ -0,0 +1,142 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using Microsoft.AspNetCore.Http;
+using Microsoft.Bot.Schema;
+using Newtonsoft.Json;
+using Twilio.Exceptions;
+using Twilio.Rest.Api.V2010.Account;
+using Twilio.Security;
+
+namespace Microsoft.Bot.Builder.Adapters.Twilio
+{
+    public static class TwilioHelper
+    {
+        /// <summary>
+        /// Extracts attachments (if any) from a twilio message and returns them in an Attachments array.
+        /// </summary>
+        /// <param name="message">The TwilioEvent message.</param>
+        /// <returns>An Attachments array with the converted attachments.</returns>
+        public static List<Attachment> GetMessageAttachments(TwilioEvent message)
+        {
```

Comment:
As _TwilioEvent_ might be _null_, shouldn't we check it? In case it's null, shouldn't we return an empty _List<Attachment>_? Or shouldn't we return _**null**_?

Reply:
Same as below, I favor null